ext/node/ops/crypto/mod.rs

```diff
@@ -64,6 +65,7 @@ pub async fn op_node_check_prime_async(
   Ok(spawn_blocking(move || primes::is_probably_prime(&num, checks)).await?)
 }
 
+// TODO(bartlomieju): blocked on `op2` supporting returning a future
```

Comment:
This should be supported. What are you getting as an error?

Reply:
Ahh. Remove the `'static` here -- it's unnecessary.

Reply:
I'm gonna leave it for now because it's also handling buffer - I'll wait until the problem with "ArrayBufferView" is fixed.